src/resources/js/pages/sessions/test-cases/show.vue

```diff
@@ -48,6 +49,7 @@
                         </thead>
                         <tbody>
                             <tr v-for="testRun in testRuns.data">
+                                <td>{{ testRun.id }}</td>
```

Comment:
```suggestion
                                <td>#{{ testRun.id }}</td>
```

I worry that having numerical IDs will make it seem like runs will always go in order, so I think making it `#123` will be clearer that it's not necessarily incremental.

Reply:
I think saying which UUID is difficult because is a huge number, now with the users group and sharing session will be easier to say check the test run 3, and etc. 
We could also remove the ID on the top collumn and replace by # so we just make reference as number and not a ID.